# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1307596234 25200
#      Wed Jun 08 22:10:34 2011 -0700
# Branch stable
# Node ID ee3a60f254e177dc6172cd120c3962d39d4a6f81
# Parent  a2eb942dee338f6a5ed742e262387c8db53ebf69
maint: Remove built file ChangeLog with distclean target

* Makefile.am: Remove built file ChangeLog with distclean target

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -111,17 +111,17 @@ noinst_SCRIPTS = run-octave
 
 INFO_FILES = \
   AUTHORS \
   BUGS \
   INSTALL.OCTAVE
 
 CLEANFILES = $(bin_PROGRAMS) $(bin_SCRIPTS) $(BUILT_SOURCES)
 
-DISTCLEANFILES = $(INFO_FILES) .gdbinit 
+DISTCLEANFILES = $(INFO_FILES) ChangeLog .gdbinit 
 
 CONFIG_FILES = @ac_config_headers@ @ac_config_files@
 
 nodist_octinclude_HEADERS = config.h
 
 all-local: $(bin_PROGRAMS) $(bin_SCRIPTS) $(noinst_SCRIPTS) $(INFO_FILES) .gdbinit
 	@echo ""
 	@echo "Octave successfully built.  Now choose from the following:"
